mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -20,12 +20,13 @@
 package org.apache.iceberg.mr;
 
 import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.FileFormat;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.SchemaParser;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.expressions.Expression;
 
-public class InputFormatConfig {
+public final class InputFormatConfig {
```

Comment:
Why make this final?

Reply:
It was showing some compile warnings before, but not now.
Removed the change